packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -916,8 +916,8 @@ class WebpackCLI implements IWebpackCLI {
         ],
         multiple: false,
         description:
-          "Sets process.env.NODE_ENV to the specified value. (Currently an alias for `--node-env`)",
-        helpLevel: "minimum",
+          "Sets process.env.NODE_ENV to the specified value. (Currently an alias for `--node-env`).",
+        helpLevel: "verbose",
```

Comment:
Exclude this, because I think it was a mistake to implement this, so let's hide it